packages/ckeditor5-clipboard/tests/dragdrop.js

```diff
@@ -2505,4 +2540,39 @@ describe( 'Drag and Drop', () => {
 			.trim()
 			.replace( />\s+</g, '><' );
 	}
+
+	function CustomTitle( editor ) {
```

Comment:
Out of curiosity, why is this tested against `CustomTitle`, and not [Title](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/features/title.html) plugin?